doc/config.rst

```diff
@@ -295,6 +295,21 @@ General configuration
 
    .. versionadded:: 1.3
 
+.. confval:: manpages_url_pattern
+
+   A URL to cross-reference :rst:role:`manpage` directives. If this is
+   defined to ``https://manpages.debian.org/{manpage_full}``, the
+   :literal:`:manpage:`man(1)`` role will like to
+   <https://manpages.debian.org/man(1)>. The patterns available are:
+
+     * ``manpage_name`` - the manpage name (``man``)
+     * ``manpage_section`` - the manpage section (``1``)
+     * ``manpage_full`` - the whole string (``man(1)``)
```

Comment:
I feel `manpage_` suffix is a bit verbose. It is obvious that these variables are related with manpage, because this settings is used for URL of manpages.

Reply:
sure, but internally, the `node` parameter would be ambiguous otherwise. since i just pass the whole `node` attributes to the formatter, i figured it was better to have unambiguous parameters.

now, i could expand that formatter to have an explicit dictionary with only those values, but that didn't seem very useful. i will do that, however, if you insist on changing the formatter...

Reply:
I feel `manpage_` prefix is also too verbose for node attributes. `manpage_node['name']` and `manpage_node['section']` are much simpler for me.
What do you think?

Reply:
er. well i explained this earlier: i thought having a `name` parameter in the node would be ambiguous. but i don't know how the parsers work - if you're saying it's unambiguous i'm happy to fix this!

Reply:
I don't think so. Usually, we know the type of nodes when we manipulate them. For example, `ManpageLink` transform searches manpage node using `node.traverse()`.  And each `visit_*` methods of writer also know the type of node.
So I feel `manpage_name` of manpage node is verbose. We already know the node represents a reference for manpage.
I understand `any_node['name']` is ambiguous. But we would not access in such way.

In addition, I feel the format params of `manpages_url` is verbose.

Reply:
understood, i'll fix this then! :)